codalab/model/tables.py

```diff
@@ -228,3 +228,18 @@
   Index('user_user_name_index', 'user_name'),
   sqlite_autoincrement=True,
 )
+
+# Store information about users' questions or feedback.
+chat = Table(
+  'chat',
+  db_metadata,
+  Column('id', Integer, primary_key=True, nullable=False),
+  Column('time', DateTime, nullable=False),  # When did the user send this query?
+  Column('date', String(63), nullable=False),  # (e.g., 2015-09-11)
+  Column('user_id', String(63), nullable=True),  # Who did it?
+  Column('user_name', String(63), nullable=True),  # Who did it?
```

Comment:
We shouldn't log user_name explicitly, since we will have the user_names to this DB and be able to join shortly.